packages/mdc-button/_mixins.scss

```diff
@@ -55,6 +55,14 @@
 
 @mixin mdc-button-ripple($ripple-config) {
   &:not(:disabled) {
+    // TODO(yiranm): Move base-color and tap-highlight-color into ripple
```

Comment:
Currently this code is only needed because we're always relying on `base-color` in the ripple config map (we're using it for `$tap-highlight-color` below).

However, Yiran and I did a brief dive to investigate why we're even setting `$tap-highlight-color` specifically here, when ripple ordinarily sets it to fully transparent. `-webkit-tap-highlight-color` only affects behavior of links on mobile, so this results in `<a class="mdc-button">` elements having a more stark effect upon being tapped on mobile browsers. However, this also makes them behave inconsistently with all other MDC Buttons, whereas removing the `-webkit-tap-highlight-color` here would make them completely consistent. We already have focused and active states for CSS-only buttons, so the buttons should still appear responsive to user interaction.

IMO the ideal scenario here is we could just switch from using `base-color` to `theme-style` above (which already expects a property from the theme map), but we're thinking of checking with someone more experienced in a11y to be sure that removing `-webkit-tap-highlight-color` here is okay first.